tests/snippets/test_abc.py

```diff
@@ -0,0 +1,39 @@
+import abc
```

Comment:
Did you copy this file from cpython? Please note that cpython has its own test suite for the standard library. We should probably pick that one.

Reply:
Yes I did copy it from cpython. Eventually we should use cpython test suite, but that requires unittest to work, which It won't at this stage.

Reply:
unittest is an excellent candidate to port first. This I tried some time back, but too many things were broken at that time.